src/custom/pages/Claim/InvestmentFlow/InvestOption.tsx

```diff
@@ -258,7 +273,7 @@ export default function InvestOption({ approveData, claim, optionIndex }: Invest
                 )}
               </span>
               <StyledNumericalInput
-                onUserInput={onInputChange}
+                onUserInput={(value) => setTypedValue(value)}
```

Comment:
I guess it doesn't work with this suggestion because `value` types are different?

```suggestion
                onUserInput={setTypedValue}
```

Reply:
Turns out it works! Tried it out locally.

Will accept my own suggestion :)